include/ed/plugin.h

```diff
@@ -32,13 +32,13 @@ class Plugin
     virtual ~Plugin() {}
 
     // Old
-    virtual void configure(tue::Configuration config) {}
+    virtual void configure(tue::Configuration /*config*/) {}
     virtual void initialize() {}
-    virtual void process(const WorldModel& world, UpdateRequest& req) {}
+    virtual void process(const WorldModel& /*world*/, UpdateRequest& /*req*/) {}
 
     // New
-    virtual void initialize(InitData& init) {}
-    virtual void process(const PluginInput& data, UpdateRequest& req) {}
+    virtual void initialize(InitData& /*init*/) {}
+    virtual void process(const PluginInput& /*data*/, UpdateRequest& /*req*/) {}
```

Comment:
Why keep those comments?

Reply:
A bit as a guideline...

Reply:
Looks weird. But if you want to keep it, I'm okay with it